### PR TITLE
Use IE GPU rendering settings

### PR DESCRIPTION
**Pull request by silfumus**
Referring to: https://github.com/RadarNyan/CefEOBrowser/issues/2#issuecomment-415605090
By default, EO uses IE registry settings to decide whether to use hardware acceleration.
I think it's better to also use it on CEF, since you wouldn't need to add a separate setting.

### CODE REVIEW

**Comment by RadarNyan** [comment]
Thanks for your contribution, but I'd like to reject this for the following reasons:

1. As you know, this setting is stored in Windows Registry.
2. If user doesn't know to click "Apply" button on that setting, the registry key won't be written at all (It doesn't save automatically with other settings)
3. That checkbox is somewhat unreliable to reflect the real status, which can confuse user.
4. That option is **off by default**, which is not acceptable.

### TL;DR, Windows Registry isn't something I want to mess about.

Besides, IMO disabling GPU should be considered a "hack" rather than a normal option, at least for now it breaks some visual effects and creates some visual artifacts in-game, only people understand the consequences should be doing so.

I implement the file check in a way that even a file like ```nogpu.txt``` would work, so that shouldn't be very hard to do. Though I need to add that in the Readme file once I decide to make a stable release.
